### DISCUSSION TITLE
Disable Guest accounts that have not logged on for 90 days

### DISCUSSION THREAD

**Discussion by DavidBrisk** [KelvinTegelaar/CIPP#1284] [Q&A] [answered]
Hi,

We set the in standards to "Disable Guest accounts that have not logged on for 90 days" and now I see in the logs "Disabled guests accounts with a login longer than 30 days ago."

Is there something I missed?

Thanks

**Comment by KelvinTegelaar** [comment, marked as answer]
Typo :) The code is 90 days. I fixed this in the dev version so will be in the next release.

**Comment by DavidBrisk** [comment]
Thank you!
